docs/partials/index.md

```diff
@@ -67,6 +67,25 @@ Aside from the standard hyperlink use-case, the `auro-hyperlink` element is inte
 
 </auro-accordion>
 
+## Unsupported types
```

Comment:
Need to run a `build:release` in order for the changes to show up on the localhost

Reply:
During a remote build/release this line is run

https://github.com/AlaskaAirlines/auro-hyperlink/blob/dff0c54009e530d081a482316208f299b27ce8fb/package.json#L166

Then these assets are committed back to the repo

https://github.com/AlaskaAirlines/auro-hyperlink/blob/dff0c54009e530d081a482316208f299b27ce8fb/package.json#L116-L123

Because of this, we are not required to run a build locally to commit these resources.